terraform/gcp/modules/monitoring/infra/alerts.tf

```diff
@@ -332,3 +332,89 @@ resource "google_monitoring_alert_policy" "k8s_container_cpu_allocatable_utiliza
   notification_channels = local.notification_channels
   project               = var.project_id
 }
+
+### Redis Alerts
+
+# Redis Memory Usage > 90%
+resource "google_monitoring_alert_policy" "redis_memory_usage" {
+  # In the absence of data, incident will auto-close in 7 days
```

Comment:
Do we want to remove this? This incident shouldn't auto close or else we risk not proactively increasing memory.

Reply:
I can get rid of it, but all of our gcp alerts have this. IIUC it would immediately retrigger if it closed out. Thoughts?

Reply:
As long as it retriggers, that’s good! Just want to make sure it surfaces again.